packages/babel-plugin-transform-react-constant-elements/src/index.js

```diff
@@ -1,4 +1,9 @@
-export default function transformReactConstantElement({ types: t }) {
+export default function transformReactConstantElement({ types: t }, options) {
+  const { allowMutablePropsOnTags } = options;
+  const isAllowMutablePropsOnTagsAnArray = Array.isArray(
```

Comment:
This and `isAllowMutablePropsOnTagsNotNull ` are also cases where I don't think a change is needed.